compiler/rustc_target/src/spec/mod.rs

```diff
@@ -1085,6 +1085,34 @@ impl ToJson for CodeModel {
     }
 }
 
+#[derive(Clone, Copy, PartialEq, Hash, Debug)]
+pub enum FloatAbi {
+    Soft,
+    Hard,
```

Comment:
Some documentation about what each variant mean would be great ;-)

Reply:
Not sure what to say here other than "use softfloat ABI" vs "use hardfloat ABI", which seems a bit pointless.

Reply:
We could hypothetically say something about what "soft float" and "hard float" mean, but I think that is something we should consider as a more general topic.